gems/ruby-saml/OSVDB-124991.yml

```diff
@@ -0,0 +1,13 @@
+---
+gem: ruby-saml
+osvdb: 124991
+url: https://github.com/onelogin/ruby-saml/pull/225
+title: Ruby-Saml Gem is vulnerable to XPath Injection
+date: 2015-07-09
```

Comment:
Let's make this 2015-04-29, as that's when https://github.com/onelogin/ruby-saml/commit/9853651b96b99653ea8627d757d46bfe62ab6448 was committed.